packages/angular_devkit/core/src/jobs/create-job.ts

```diff
@@ -72,6 +72,28 @@ export interface JobHandlerWithExtra<
 }
 
 
+export function renameJob<I extends JsonValue, O extends JsonValue>(
```

Comment:
This doesn't seem used anywhere.

It also does far more than just rename a job.

Reply:
It's used in the Tasks API. I'll rename it.

Reply:
_And_ move it to the original Jobs PR (not this one).